server/simulation/random.js

```diff
@@ -22,11 +26,12 @@ const randomDroneModel = () => {
   return `${randomManufacturerName()} ${randomModelName()}`;
 };
 
-/**
- * Returns a random rating between 1.0 and 5.0
- *
- * @returns {Number} A floating point number representing a rating between 1.0 and 5.0
- */
+-/**		
```

Comment:
Your pull request modifies this unrelated piece of code.
Please make sure your pull request does not modify anything except that which you are trying to fix.

It's a good idea to always check the view of the changes you made before making the pull request to make sure there are no intended side effects.